changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,31 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "6c5c31e1-51bb-45c2-b769-4928b4977187",
+          "ApiChange": "Resource",
+          "ChangedApiName": "externalIdentitiesPolicy",
+          "ChangeType": "Addition",
+          "Description": "Added the [externalIdentitiesPolicy](https://docs.microsoft.com/en-us/graph/api/resources/externalIdentitiesPolicy?view=graph-rest-beta) resource",
```

Recommending a little more info to the customer.
```suggestion
          "Description": "Added the [externalIdentitiesPolicy](https://docs.microsoft.com/en-us/graph/api/resources/externalIdentitiesPolicy?view=graph-rest-beta) resource and associated methods. The external identity policy represents the tenant-wide policy that controls whether external users can leave an Azure AD tenant via self-service controls.",
```